### PR TITLE
ASSETS Generator

### PR DESCRIPTION
**Pull request by jaycwb**

### CODE REVIEW

File: moa/src/main/java/moa/streams/generators/AssetNegotiationGenerator.java [hunk 1]
```diff
@@ -0,0 +1,429 @@
+package moa.streams.generators;
+
```
Comment (abifet) [inline moa/src/main/java/moa/streams/generators/AssetNegotiationGenerator.java]: Please add license header

File: moa/src/main/java/moa/streams/generators/AssetNegotiationGenerator.java [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+    public FloatOption noisePercentage = new FloatOption("noise", 'n',
+	    "% of class noise.", 0.05, 0.0, 1.0f);
+
+    public IntOption instanceRandomSeedOption = new IntOption(
+	    "instanceRandomSeed", 'i',
+	    "Seed for random generation of instances.", 1);
+
+    ///////////////
+    // INTERNALS //
+    ///////////////
```
Comment (abifet) [inline moa/src/main/java/moa/streams/generators/AssetNegotiationGenerator.java]: Please use standard Java comments

**Comment by jaycwb** [comment]
Added License header and changed the comments format.

Cheers,

Jean

**Comment by abifet** [comment]
+1

**Pull request merged**